src/program/services/scrapers/__init__.py

```
@@ -85,24 +95,25 @@ def run(self, item: MediaItem) -> Generator[MediaItem, None, None]:
-        item.set("scraped_times", item.scraped_times + 1)
+        item.set("scraped_times", (item.scraped_times or 0) + 1)
 
-        yield item
+        yield RunnerResult(media_items=[item])
 
-    def scrape(self, item: MediaItem, verbose_logging=True) -> Dict[str, Stream]:
+    def scrape(
+        self,
+        item: MediaItem,
+        verbose_logging=True,
+        ranking_overrides: Dict[str, list[str]] | None = None,
```

Ruff (0.14.6)

Undefined name `Dict`

(F821)